### PR TITLE
refactor the project structure

### PR DESCRIPTION
**Pull request by ts-nitish-a-jain**
This PR only aims at refactoring the project structure to create base for foundation work in the subsequent PRs (which will target the code infrastructure)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,12 +46,12 @@ plugins {
 
 allprojects {
   repositories {
-    artifactoryRelease()
-    artifactorySnapshot()
-    mobileLibs()
+//    artifactoryRelease()
+//    artifactorySnapshot()
```
Comment (corycaywood) [inline build.gradle]: We can remove this and `artifactoryRelease`

File: build.gradle [hunk 2]
```diff
@@ -46,12 +46,12 @@ plugins {
...
+//    jitpack()
     google()
     jcenter()
-    jitpack()
+//    mobileLibs()
```
Comment (corycaywood) [inline build.gradle]: This one can be removed too.

File: build.gradle (hunk 1 above)
Comment (ts-nitish-a-jain) [reply build.gradle]: Got it. Thanks. Will remove in the upcoming one.

File: build.gradle (hunk 2 above)
Comment (ts-nitish-a-jain) [reply build.gradle]: Got it. Thanks. Will remove in the upcoming one.

**Pull request merged**